{
    "docstring": "",
    "name": "GeoIP2Exception",
    "namespace": "django.contrib.gis.geoip2.base.GeoIP2Exception",
    "module_name": "django.contrib.gis.geoip2.base",
    "package_name": "",
    "methods": []
}